lib/Bio/EnsEMBL/DataCheck/index.json

```diff
@@ -47,24 +59,24 @@
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::ArraysHaveProbes",
       "description" : "All arrays are associated with probes."
    },
-   "AssemblyException" : {
+   "CoreForeignKeys" : {
```

Comment:
I'll make it so that this file is generated in alphabetical order, to prevent the situation here, where it looks like there are lots of changes, but really it's just that elements have been arbitrarily reordered.